src/dialogs/more-info/controls/more-info-cover.ts

```diff
@@ -0,0 +1,151 @@
+import { css, CSSResult, html, LitElement, TemplateResult } from "lit";
+import { customElement, property } from "lit/decorators";
+import { attributeClassNames } from "../../../common/entity/attribute_class_names";
+import { featureClassNames } from "../../../common/entity/feature_class_names";
+import "../../../components/ha-attributes";
+import "../../../components/ha-cover-tilt-controls";
+import "../../../components/ha-labeled-slider";
+import {
+  CoverEntity,
+  isTiltOnly,
+  supportsSetPosition,
+  supportsSetTiltPosition,
+} from "../../../data/cover";
+import { HomeAssistant } from "../../../types";
+
+const FEATURE_CLASS_NAMES = {
+  4: "has-set_position",
+  16: "has-open_tilt",
+  32: "has-close_tilt",
+  64: "has-stop_tilt",
+  128: "has-set_tilt_position",
+};
+
+@customElement("more-info-cover")
+class MoreInfoCover extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false }) public stateObj!: CoverEntity;
+
+  protected render(): TemplateResult {
+    if (!this.stateObj) {
+      return html``;
+    }
+
+    const _isTiltOnly = isTiltOnly(this.stateObj);
+
+    return html`
+      <div class=${this._computeClassNames(this.stateObj)}>
+        <div class="current_position">
+          <ha-labeled-slider
+            .caption=${this.hass.localize("ui.card.cover.position")}
+            pin=""
+            .value=${this.stateObj.attributes.current_position}
+            .disabled=${!supportsSetPosition(this.stateObj)}
+            @change=${this._coverPositionSliderChanged}
+          ></ha-labeled-slider>
+        </div>
+
+        <div class="tilt">
+          ${supportsSetTiltPosition(this.stateObj)
+            ? // Either render the labeled slider and put the tilt buttons into its slot
+              // or (if tilt position is not supported and therefore no slider is shown)
+              // render a title <div> (same style as for a labeled slider) and directly put
+              // the tilt controls on the more-info.
+              html`
+                <ha-labeled-slider
+                  .caption=${this.hass.localize("ui.card.cover.tilt_position")}
+                  pin=""
+                  extra=""
+                  .value=${this.stateObj.attributes.current_tilt_position}
+                  @change=${this._coverTiltPositionSliderChanged}
+                >
+                  <ha-cover-tilt-controls
+                    .hass=${this.hass}
+                    slot="extra"
+                    .hidden=${_isTiltOnly}
```

Comment:
Does this do anything?

Reply:
You mean the `hidden` condition? That is there in case a cover only supports tilting but not up/down. In that case, the tilt buttons should not be shown above the tilt slider, but instead they will be shown as part of the state card (main buttons). That is the existing behavior.

Reply:
Where is the hidden property consumed then? I don't see it...

Reply:
It is not "consumed". It is the regular HTML `hidden` attribute. It was only by mistake a property here. Changed that.

Reply:
Please use a ternary for rendering instead of this attribute